### PR TITLE
Add operational profiles study

### PR DESCRIPTION
**Pull request by LorenzoMinto**
Wiki: https://github.com/brave/brave-browser/wiki/Operational-Patterns

### CODE REVIEW

File: seed/seed.json [hunk 1]
```diff
@@ -542,6 +542,50 @@
...
+                        "disable_feature": ["FederatedLearningOperationalProfiling"]
+                    }
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "min_version": "90.1.27.68",
```
Comment (LorenzoMinto) [inline seed/seed.json:584]: @moritzhaller could you double-check this?

**Comment by kjozwiak** [comment]
@moritzhaller mind taking a look at https://github.com/brave/brave-variations/pull/65#discussion_r654565280?

File: seed/seed.json (hunk 1 above)
Comment (goodov) [reply seed/seed.json:584]: should be ```91.x.x.x```
https://github.com/brave/brave-browser/blob/1.27.x/package.json

File: seed/seed.json [hunk 2]
```diff
@@ -542,6 +542,50 @@
...
+                            "value": "30"
+                        }, 
+                        {
+                            "name": "simulate_local_training_step_duration_in_minutes",
+                            "value": "5"
+                        },
+                        {
+                            "name": "collection_id_lifetime_in_days",
+                            "value": "30"
+
```
Comment (goodov) [inline seed/seed.json:567]: unnecessary empty line

File: seed/seed.json [hunk 3]
```diff
@@ -542,6 +542,50 @@
                 "channel": ["NIGHTLY", "BETA", "RELEASE"],
                 "platform": ["WINDOWS", "MAC", "LINUX", "ANDROID"]
             }
+        },
+        {
+            "name": "OperationalProfilesCollectionStudyBeta",
```
Comment (moritzhaller) [inline seed/seed.json:547]: nit: rename to `OperationalProfilesCollectionStudy`

File: seed/seed.json [hunk 4]
```diff
@@ -542,6 +542,50 @@
...
+                },
+                {
+                    "name": "Default",
+                    "probability_weight": 0
+                }
+            ],
+            "filter": {
+                "min_version": "90.1.27.68",
+                "channel": ["BETA"],
+                "platform": ["WINDOWS", "MAC", "IOS", "ANDROID"],
```
Comment (moritzhaller) [inline seed/seed.json:586]: remove `IOS` as not supported by variations currently

**Comment by kjozwiak** [comment]
Going to remove `brave/uplift-approvers` from the review list till this is finally ready 👍 Please feel free to re-add `brave/uplift-approvers` when we're ready to merge into `main`.

File: seed/seed.json [hunk 5]
```diff
@@ -942,6 +942,44 @@
                 "channel": ["RELEASE", "BETA", "NIGHTLY"],
                 "platform": ["WINDOWS", "MAC", "LINUX"]
             }
+        },
+        {
+            "name": "OperationalPatternsCollectionStudyBeta",
```
Comment (goodov) [inline seed/seed.json:947]: Beta or Nightly?
```"channel": ["NIGHTLY"],```

Comment (LorenzoMinto) [reply seed/seed.json:947]: Good point. We want to run it on Nightly first. I'm just going to remove the channel from the study title so we don't have to rename it each time

**Review by moritzhaller — APPROVED** [review]

**Review by goodov — APPROVED** [review]

**Review by goodov — APPROVED** [review]

**Pull request merged**